presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -110,7 +97,8 @@ protected QueryRunner createQueryRunner()
             queryRunner.createTestFunctionNamespace("example", "example");
             queryRunner.getMetadata().getFunctionAndTypeManager().addUserDefinedType(MOOD_ENUM);
             queryRunner.getMetadata().getFunctionAndTypeManager().addUserDefinedType(COUNTRY_ENUM);
-            queryRunner.getMetadata().getFunctionAndTypeManager().addUserDefinedType(PERSON);
+
+            queryRunner.execute("CREATE TYPE testing.type.person AS (first_name varchar, last_name varchar, age tinyint, country testing.enum.country)");
```

Comment:
Let's keep this as is to be consistent with the other type initialization. It's probably a good follow up to add syntax support for creating enum types. cc @kaikalur suggestions on syntax?